### ISSUE TITLE
Crash on Android 9 (SDK 28) of Type java.lang.NoSuchMethodError

### ISSUE THREAD

**Issue by moazelsawaf** [nizwar/screen_capture_event#8]
Hello, 

The following crash occur in Android 9 (SDK 28) Specifically in Huawei Nova 3i:

```
java.lang.NoSuchMethodError: 
  at id.nizwar.screen_capture_event.ScreenCaptureEventPlugin$3.<init> (ScreenCaptureEventPlugin.java:4)
  at id.nizwar.screen_capture_event.ScreenCaptureEventPlugin.updateScreenRecordStatus (ScreenCaptureEventPlugin.java:85)
  at id.nizwar.screen_capture_event.ScreenCaptureEventPlugin.onMethodCall (ScreenCaptureEventPlugin.java:120)
  at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage (MethodChannel.java:17)
  at io.flutter.embedding.engine.dart.DartMessenger.invokeHandler (DartMessenger.java:18)
  at io.flutter.embedding.engine.dart.DartMessenger.lambda$dispatchMessageToQueue$0 (DartMessenger.java:20)
  at io.flutter.embedding.engine.dart.DartMessenger.lambda$dispatchMessageToQueue$0$DartMessenger (DartMessenger.java)
  at io.flutter.embedding.engine.dart.-$$Lambda$DartMessenger$TsixYUB5E6FpKhMtCSQVHKE89gQ.run (-.java:12)
  at android.os.Handler.handleCallback (Handler.java:907)
  at android.os.Handler.dispatchMessage (Handler.java:105)
  at android.os.Looper.loop (Looper.java:216)
  at android.app.ActivityThread.main (ActivityThread.java:7625)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:524)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:987)
```

**Comment by overdrowe** [comment]
@moazelsawaf Hey, have you found a solution?

**Comment by moazelsawaf** [comment]
@overdrowe 

To be honest I have stopped working in the project that I was using this package inside, so I have not tried again with it.

**Comment by nizwar** [comment]
Fixed on 1.1.0
